packages/documentation/docs/controls/_dropdown_styleguide.md

```diff
@@ -0,0 +1,43 @@
+Dropdowns are containers into which you can insert elements. The dropdown containers do not function independently; they must be opened via a dropdown button, for example. The dropdown container contains items that offer the user a specific selection. These items can be customized depending on their intended use.
+
+Caution, only the dropdown container and its items are described here! For information about the element that opens it, read the article about the dropdown button or the split button.
+
+![Overview](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=2353-2278&mode=design&t=OVHeXvLZYLkP2CzN-4)
+
+1. Dropdown Container
+2. Item
+3. Header
+4. Quick Actions
+5. Submenu Item
+6. Checked
+7. Separator
+8. Icon
+9. Label
+
+## Options
+
+- **Header:** Add an header for the dropdown container. 
```

Comment:
keep an eye on the usage of an vs. a, and ending every item with a full stop (.)